### ISSUE TITLE
alt accounts made with new local.dat files send settings to main account

### ISSUE THREAD

**Issue by gunnho** [Healix/Gw2Launcher#10]
i have a main account never used in gw2launcher that adopts the settings of alt accounts made with launcher. i created 3 other accounts, created new local.dat files for each of them through launcher, edited them (without -sharearchive active) to have low graphics and sound muted and all was well. i opened them and tested them and all had low gfx and muted.

then i opened my main account without the gw2launcher and it adopted the low settings and mute. i changed all this back to high settings the way i want to play my main account, then used gw2launcher to open my alts and they all had high settings, i re-edited the alts through launcher in normal mode without -sharearchive back to low. closed and opened each one and all was well.

then i booted my main without gw2launcher and it once again had all low settings.

i thought the local.dat file stored it's own settings for gfx and every account should have individual settings?

**Comment by Healix** [comment]
Check the path of each Local.dat and GFXSettings.xml file. If the path has Gw2Launcher near the ends, it's okay, whereas if it has Guild Wars 2, it's using GW2's file and you'll want to select a different one. These can be configured per account by right clicking > Edit > Local.dat (and Graphics under it).

Note that if these files aren't configured when creating an account, Gw2Launcher will default to using GW2's normal files as long as they haven't already been linked to an account.

**Comment by gunnho** [comment]
awesome, i thought i had gone through all the settings but now i see how to fix this, thanks.

**Comment by JinxLuckless** [comment]
I am having this same issue, but I'm not able to get it working even when following the steps above as well as I'm able.

I have made sure that my accounts each have their own local.dat file and that it's in the GW2Launcher folder, and I've also loaded each one with Selected--> Launch (Normal) to change settings. But even so, if I change the graphics settings for any of them, it changes the settings for all of them. However, the sound settings *won't* be copied over, which I tested with 'mute when in background.'

The local.dat files are definitely different for each of them-- for instance, my second account's .dat is located here: AppData\Roaming\Gw2Launcher\data\2.dat, and my third account has 3.dat.

The steps I've tried doing:
1) Doublecheck that account 1 has its own .dat file. Right click on account 1, and choose Selected--> Launch (Normal).
2) Change graphics settings while logged into a character and set it to not mute GW2 when it's in the background. Hit escape, then 'exit to desktop.'
3) Reload the same account using a left click on the account name in GW2Launcher, to be sure the settings saved corrrectly, which they always have. I hit escape, then 'exit to desktop.'
4) Make sure account 2 has its own .dat file. Right click on account 2, and choose Selected--> Launch (Normal). It will have the same graphics settings that I just set for account 1, but 'mute when in background' will be unchecked.
5) Change graphics settings so they're different than account 1's settings, and change it to mute GW2 when it's in the background. Hit escape, then 'exit to desktop.'
6) Left click on account 2 to relaunch it, to be sure the settings saved. Again, successful. Hit escape, then 'exit to desktop.'
7) Left click on account 1 to launch it. It will have the graphics settings I just picked for account 2, but it keeps its 'don't mute when in background' setting.

I'm using the beta version of the launcher, the one from 18 hours ago that's 2264 kb in size. In case it matters, my GW2 management settings are to use virtual mode, enable localized execution is checked, and 'exclude addons when generating folders' is unchecked. Desired level is set to Full, and synchronizing is set to Core.
